bin/main.rb

```diff
@@ -1,81 +1,100 @@
 #!/usr/bin/env ruby
-# Ask the players for their names
-puts 'Enter player 1 name:'
-player1 = gets.chomp
+require 'pry'
+require_relative '../lib/player.rb'
+require_relative '../lib/table.rb'
+require_relative '../lib/colors.rb'
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/MethodLength, Metrics/AbcSize
 
-puts 'Enter player 2 name:'
-player2 = gets.chomp
+def game_initializer
+  players_objects = []
+  puts 'Enter Player 1 name please:'.blue
+  player1 = gets.chomp
+  players_objects << player1
+  puts 'Enter Player 2 name please'.blue
+  player2 = gets.chomp
 
-# Display players names
```

Comment:
- [ ] optional => Here, it would be nice to add a validation for not allowing the user to press enter without adding the name (right now it is possible).

![image](https://user-images.githubusercontent.com/61479648/95267909-a7a26680-080c-11eb-80c2-49b146fba82e.png)

- [ ] optional => I think you could optimize this slightly. You do not need the player objects array, player one variable. Check the way it is on the print screen below:

![image](https://user-images.githubusercontent.com/61479648/95267996-d6b8d800-080c-11eb-8af7-5453c1a3abe4.png)

:arrow_double_up: 

While it is shorter, it also saves a little more memory.